axlearn/common/gradient_accumulation.py

```diff
@@ -172,12 +167,54 @@ def fwd_helper(
                 otherwise None.
             """
             minibatch_size = _compute_minibatch_size(inputs["input_batch"], steps=steps)
+
+            def reshape_for_scan(x: Tensor):
+                """Helper function that adds a minibatch dimension while evenly dividing
+                batches across gradient accumulation iterations.
+
+                Input dimension is [Global logical Batch Size, Sequence], this first reshaped to
```

Comment:
nit -- 
```suggestion
                Input dimension is [global_logical_batch_size, seq_len], this first reshaped to
```

ditto for below.